src/glue/connection.ts

```diff
@@ -16,27 +16,88 @@ export enum ConnectionType {
+   *    JDBC - Designates a connection to a database through Java Database Connectivity (JDBC).
+   *    KAFKA - Designates a connection to an Apache Kafka streaming platform.
+   *    MONGODB - Designates a connection to a MongoDB document database.
+   *    NETWORK - Designates a network connection to a data source within an Amazon Virtual Private Cloud environment (Amazon VPC).
+   * 
+   * @see [AWS::Glue::Connection ConnectionInput](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-connection-connectioninput.html#cfn-glue-connection-connectioninput-connectiontype)
+   */
   readonly connectionType: ConnectionType;
+  /**
+   * A description for the Connection.
+   */
   readonly description?: string;
+  /**
+   * The name of the connection. Connection will not function as expected without a name.
+   * 
+   * @see [AWS::Glue::Connection ConnectionInput](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-connection-connectioninput.html#cfn-glue-connection-connectioninput-name)
+   */
   readonly name?: string;
+  /**
+   * List of Key/Value pairs defining the properties of the Connection 
+   * Available Properties:
+   *    CatalogId - The ID of the data catalog to create the catalog object in. Currently, this should be the AWS account ID.
+   *    ConnectionInput - The connection that you want to create.
+   * 
+   * @see [AWS::Glue::Connection Properties](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-connection.html#Properties)
+   */
   readonly properties?: {[key: string]: string};
+  /**
+   * Existing Security Group to assign to the Connection. If none is provided a new Security Group will be created. 
+   */
   readonly securityGroups?: ISecurityGroup[];
+  /**
+   * Options for selection of subnets from the VPC to attach to the Connection 
+   * 
+   * @see [CDK SubnetSelection](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ec2.SubnetSelection.html)
+   */
   readonly subnets?: SubnetSelection;
+  /**
+   * VPC to attach to the Connection 
+   * 
+   * @see [IVpc Interface](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ec2.IVpc.html)
+   */
   readonly vpc?: IVpc;
 }
 
+/**
+ * Creates a resource specifying a Glue Connection to a data source.
+ * 
+ * @see [AWS::Glue::Connection](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-connection.html)
+ */
 export class Connection extends Resource implements IConnectable {
   // Internal properties
   private readonly _matchCriteria: string[] = [];
   private readonly _properties: {[key: string]: string} = {};
 
   // Input properties
-  public readonly connectionType: ConnectionType;
-  public readonly description?: string;
-  public readonly name?: string;
-  public readonly securityGroups: ISecurityGroup[];
-  public readonly subnets?: SubnetSelection;
-  public readonly vpc?: IVpc;
+  /**
```

Comment:
Indentation here looks off.

Reply:
Tabs vs Spaces issue. Should be fixed now